modules/ding_availability/js/ding_availability.js

```diff
@@ -54,6 +54,9 @@
               ding_availability_update_holdings(id, entity_ids);
             }
           });
+        },
+        error: function (xhr, status, error) {
```

Comment:
Am I right in assuming this is not specifically for FBS, but in case any provider fails?